src/TimeSelect.jsx

```diff
@@ -80,10 +84,36 @@ var TimePicker = React.createClass({
     var hoursMins = formattedTime.split(':');
     var newDate = new Date(this.props.value || new Date());
     newDate.setHours(hoursMins[0], hoursMins[1], 0, 0);
-
     return newDate;
   },
 
+  generateHours: function() {
+    var hours = [];
+
+    for (var hour = 0; hour < 24; hour++) {
+      if (hour < 10) {
+        hours.push('0' + hour);
+      } else {
+        hours.push(hour.toString());
+      }
+    }
+    return hours;
+  },
+
+  generateMinutes: function() {
+    var minutes = [];
+
+    for (var minute = 0; minute <= 60 - this.props.step; minute += this.props.step) {
+      if (minute < 10) {
+        minutes.push('0' + minute);
+      } else {
+        minutes.push(minute.toString());
+      }
+    }
```

Comment:
Moment isn't currently included in the project. I'm happy to add it in, but not convinced it's worth the overhead for this as it's arguably quite basic stuff.